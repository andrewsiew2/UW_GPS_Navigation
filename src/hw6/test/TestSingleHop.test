# Tests for if a path from a node to another with a single hop can be found

LoadGraph tester testing.tsv

FindPath tester BATMAN SUPERMAN