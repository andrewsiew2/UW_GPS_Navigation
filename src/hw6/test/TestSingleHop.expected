# Tests for if a path from a node to another with a single hop can be found

loaded graph tester

path from BATMAN to SUPERMAN:
BATMAN to SUPERMAN via 28